src/main/java/team/asd/javaintern/controller/TestMessageController.java

```diff
@@ -0,0 +1,31 @@
+package team.asd.javaintern.controller;
+
+import java.time.LocalDate;
+
+public class TestMessageController {
```

Comment:
@demurelebowski check this tutorial for Controller information: https://www.baeldung.com/spring-controllers#Controller